data/xml/schema.rnc

```diff
@@ -7,7 +7,12 @@ url = element url {
 }
 fixed-case = element fixed-case { MarkupText }
 tex-math = element tex-math { text }
-MarkupText = (text | b | i | url | fixed-case | tex-math )+
+sup = element sup { MarkupText }
+a = element a {
+  attribute href { xsd:anyURI },
+  MarkupText
+}
+MarkupText = (text | b | i | url | sup | a | fixed-case | tex-math )+
```

Comment:
I have added handling for `sup` in two places, but am not sure if that's everything (or that I've done it right in `latex_to_unicode.py`).

Reply:
I just remembered: One way of representing these would be to just wrap them in `<tex-math>`, because that's already handled.

So `<sup>th</sup>` could become `<tex-math>^{\textrm{th}}</tex-math>`. (This is what it would look like if the abstracts had contained this as a TeX expression in the submitted material.)

Depending on how often such tags appear, it's a trade-off between clarity (the `<sup>` version is easier to read) and not adding too many extra cases to our code.

Reply:
They're really rare, so this is probably easier. Imagine the ingestion script(s?) will need to convert any instances of `<sup>` in the xml files, but not sure where to do that

Reply:
Yes, unfortunately I'm not sure where exactly these changes belong either. We've had some weird new errors being introduced in more recent ingestions, the `&lt;&gt;` escaped tags being one of those. It would of course be best to address this problem at the source. (Tagging @mjpost @anthology-assist in case they know where those come from)